locales/en-US.yml

```diff
@@ -697,6 +697,17 @@ en-US:
       this_month: This month
       this_week: This week
       view_them: View them.
+  activerecord:
+    errors:
+      models:
+        release:
```

Comment:
I think these keys can be removed, which will make the error messages usable for all models that have these keys.

Reply:
errors.attributes.name.blank